app/src/components/Contexts/config.js

```diff
@@ -18,23 +22,35 @@ export const NETWORKS = {
         chainId : 42161
     }
 }
+
+// Contracts deployed by @ericyoondotcom with account 0xdCafA75DCAED81cba4155706E9d666112950E854 on 2023-01-26 via deploy script.
 export const ACTIVE_CONTRACTS = {
     wolvercoin : {
-        ABI : Wolvercoin,
-        address : "0x9bced5e272c4d81b55ae092f85f6fee39545686f"
+        BrownieOutput : Wolvercoin,
```

Comment:
Application Binary Interface is preferred to 'brownieOutput' as brownie can output lots of things

Reply:
Disagree, BrownieOutput contains the entire build json, not just the ABI array. So it looks weird to call `ACTIVE_CONTRACTS.wolvercoin.abi.abi`